File: petgo_app/lib/features/content/domain/report_reason.dart
/// 举报类型（Story 3.7，与后端 `ReportReason` 对齐，UPPER_SNAKE 线格式）。用户单选。
enum ReportReason {
  illegal('ILLEGAL'),
  misinfo('MISINFO'),
  inappropriate('INAPPROPRIATE'),
  harassment('HARASSMENT'),
  other('OTHER');

  const ReportReason(this.wire);

  final String wire;
}

File: petgo-backend/src/main/java/com/petgo/moderation/domain/ReportReason.java
package com.petgo.moderation.domain;

/**
 * 举报类型（Story 3.7，FR-25，落库 varchar UPPER_SNAKE）。用户单选其一。
 */
public enum ReportReason {
    ILLEGAL,        // 违法违规
    MISINFO,        // 虚假信息
    INAPPROPRIATE,  // 不当内容
    HARASSMENT,     // 骚扰
    OTHER           // 其他
}
